sklearn/datasets/descr/california_housing.rst

```diff
@@ -0,0 +1,47 @@
+.. _california_housing:
+The California Housing dataset
+================================
+Notes
+-----
+Data Set Characteristics:
+    :Number of Instances: 20640
+    :Number of Attributes: 9
+    :Attribute Information:
+        - Median House Value
+        - Median Income
+        - Housing Median Age
+        - Total Rooms
+        - Total Bedrooms
+        - Population
+        - Households
+        - Latitude
+        - Longitude
+    :Missing Attribute Values:
+        None (Missing values were taken out prior
+        to publishing of the dataset)
+               
+This data set contains a set of housing information taken 
+in the 1990 Census (in the US). The :func:`sklearn.datasets.fetch_california_housing`
+function is the data fetching / caching function that downloads the data
+archive from AT&T.

```

Comment:
What does AT&T have todo with California Housing?

Reply:
AT&T has the dataset in storage I believe.